policy/leaving-governance.md

```diff
@@ -4,13 +4,23 @@ This documents the process of leaving Electron Governance. A contributor is no l
 
 ## Offboarding Items
 
-* Update README file for the Working Group. Typically the current WG chair will submit this, but any member of the WG can submit.
+1. A governance member (Typically the WG Chair) update README file in [governance](https://github.com/electron/governance) for the WG the member is leaving.
 
-* Submit a PR in [.permissions repo](https://github.com/electron/.permissions/). Typically the current WG chair will submit this, but any member of the WG can submit.
-    * Merging this PR will automatically remove the contributor from:
-        * The WG's team on GitHub
-        * The WG's user group on Slack
+2. A governance member (Typically the WG Chair) submits a PR in [.permissions repo](https://github.com/electron/.permissions/).
+  * Merging this PR will automatically remove the contributor from:
+    * The WG's team on GitHub
+    * The WG's user group on Slack
 
-* Request an admin of Electron's Slack to change the contributor to a guest account. Message in #wg-community-safety for this. Any member of Community & Safety WG is a Slack admin.
+3. A Slack Admin demotes the member's account from Full Member to Multi-Channel Guest
```

Comment:
Should we formally specify the policy on which channels people get to stay in after being demoted to an MCG?